src/components/CommentList.jsx

```diff
@@ -0,0 +1,54 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import CommentCard from "./CommentCard";
+import { getCommentsByReviewId } from "../utils/utils";
+
+function CommentList() {
+  const { review_id } = useParams();
+  const [commentList, setCommentList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+
+  useEffect(() => {
+    getCommentsByReviewId(review_id)
+      .then((comment) => {
+        setCommentList(comment)
+        setIsLoading(false);
+      })
+  }, []);
+
+  if (isLoading) {
+    return <p>Loading...</p>;
+  }
+
+  if (!commentList) {
+    return (
+      <main >
+        <h4><u>Comments to This Review</u></h4>
+        <h3>
+          There is no comment for this review. Wanna start a conversation?
+        </h3>
+        <form>
```

Comment:
Not sure what the purpose of this input is as it doesn't do anything here.

Reply:
Yes it doesn't do anything. I just put an input box here for adding new comment which I am going to do later